sdk/servicebus/service-bus/src/connectionContext.ts

```diff
@@ -139,10 +139,11 @@ type ConnectionContextMethods = Omit<
 async function callOnDetachedOnReceivers(
   connectionContext: ConnectionContext,
   contextOrConnectionError: Error | ConnectionError | AmqpError | undefined,
-  receiverType: ReceiverType
+  receiverType: NonSessionReceiverType
 ): Promise<void[]> {
   const detachCalls: Promise<void>[] = [];
 
+  // For non-sessions batching and streaming receivers
```

Comment:
This clarification is useful in the doc comment for the method `callOnDetachedOnReceivers`

Reply:
Added